code/modules/mob/new_player/new_player_vr.dm

```diff
@@ -16,6 +16,11 @@
 		pass = FALSE
 		to_chat(src,"<span class='warning'>You have not set your scale yet. Do this on the VORE tab in character setup.</span>")
 
+	//Can they play?
+	if(!is_alien_whitelisted(src,all_species[client.prefs.species]) && (all_species[client.prefs.species].spawn_flags & SPECIES_WHITELIST_SELECTABLE))
```

Comment:
Man, this is probably a check that should have been in there in the first place for normal whitelisted species, in case someone somehow ends up with a whitelisted species saved (like if they had a diona character slot saved before they were whitelisted)

Reply:
Actually, if they had a whitelisted species, then upon opening character select screen it resets to human, so that failsafe already existed.